.github/generate-doc.py

```diff
@@ -0,0 +1,168 @@
+import glob
+import os
+import shutil
+import subprocess
+
+
+class Colors:
+    RED = '\033[91m'
+    GREEN = '\033[92m'
+    YELLOW = '\033[93m'
+    BLUE = '\033[94m'
+    RESET = '\033[0m'
+
+
+def auto_doc_installed():
+    auto_doc_cmd = os.environ.get("DOC_CMD")
```

Comment:
Same here, can we set the default command to `auto-doc` if the env var is not set?